.github/gke-workflow/xpk/tcpxo_decorator.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/src/xpk/core/workload_decorators/tcpxo_decorator.py b/src/xpk/core/workload_decorators/tcpxo_decorator.py
+index 322e574..0a71db8 100644
+--- a/src/xpk/core/workload_decorators/tcpxo_decorator.py
++++ b/src/xpk/core/workload_decorators/tcpxo_decorator.py
+@@ -163,7 +163,12 @@ def add_tcpxo_daemon_container(job_manifest):
+           {'name': 'sys', 'mountPath': '/hostsysfs'},
+           {'name': 'proc-sys', 'mountPath': '/hostprocsysfs'},
+       ],
+-      'env': [{'name': 'LD_LIBRARY_PATH', 'value': '/usr/local/nvidia/lib64'}],
++      'env': [
++          {'name': 'LD_LIBRARY_PATH', 'value': '/opt/nvidia/nccl/lib:/usr/local/cuda-12.8/targets/x86_64-local/lib:/usr/local/nvidia/lib64'},
++          {'name': 'NCCL_LIB_DIR', 'value': '/opt/nvidia/nccl/lib'},
++          {'name': 'NCCL_TUNER_PLUGIN', 'value': 'none'},
++          {'name': 'NCCL_NET_PLUGIN', 'value': '/opt/hpcx/nccl_rdma_sharp_plugin/lib/libnccl-net.so'}
++      ],
```

Comment:
I'm surprised this is needed? `add_tcpxo_daemon_container` looks like this is the environment of the sidecar container that runs the TCPXO daemon, which will presumably not have the `/opt/hpcx` and `/opt/nvidia` paths.